doc/api/repl.md

```diff
@@ -603,7 +603,9 @@ changes:
   * `eval` {Function} The function to be used when evaluating each given line
     of input. **Default:** an async wrapper for the JavaScript `eval()`
     function. An `eval` function can error with `repl.Recoverable` to indicate
-    the input was incomplete and prompt for additional lines.
+    the input was incomplete and prompt for additional lines. If a custom
+    `eval` function is provided, `callback` must be invoked to allow processing
```

Comment:
It's not clear what `callback` refers to.

Reply:
I've improved this section, lmk if it's good now.